yang/frr-isisd.yang

```diff
@@ -881,6 +881,68 @@ module frr-isisd {
           description
             "This leaf describes the state of the interface.";
         }
+
+        container adjacency-sids {
+          description
+            "This container lists the information of adjacency SID.";
+          list adjacency-sid {
+            leaf af {
+              type uint8;
+              description
+                "This leaf describes the protocol-family associated with the
+                 adjacency SID.";
+            }
+
+            leaf value {
+              type uint32;
+              description
+                "This leaf describes the value of adjacency SID.";
+            }
+
+            leaf weight {
+              type uint8;
+              description
+                "This leaf describes the weight of the adjacency SID.";
+            }
+
+            leaf protection-requested {
+              type boolean;
+              description
+                "This leaf describes if the adjacency SID must be protected.";
+            }
+          }
+        }
+
+        container lan-adjacency-sids {
```

Comment:
Put the 'container lan-adjacency-sids' definition under a new grouping called 'lan-adjacency-sids' outside and have just the 'uses lan-adjacency-sids' statement here.